editor/profile_picture.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <string>
+
+namespace editor
+{
+class ProfilePicture
+{
+public:
+    // Returns downloaded image or cached image if one exists.
+    // if not, an empty string is returned.
+    static std::string download(const std::string& pic_url);
```

Comment:
There should be a cpp code style doc.
```suggestion
    static std::string download(std::string const & pic_url);
```